src/java/org/apache/cassandra/db/ReadExecutionController.java

```diff
@@ -113,6 +123,11 @@ static ReadExecutionController forCommand(ReadCommand command)
                 throw e;
             }
         }
+        if (baseCfs.metric.topLocalReadQueryTime.isEnabled())
+        {
```

Comment:
Single line if conditions don't need braces.

Reply:
> Single line if conditions don't need braces.

In this case it is better to write braces for readability issues even that we don't need them

Reply:
It's more to follow cassandra style conventions: http://cassandra.apache.org/doc/latest/development/code_style.html `Always include braces for nested levels of conditionals and loops. Only avoid braces for single level.`

This ticket has already been resolved though. I'll close the PR.